### PR TITLE
New version: EdrawSoft.EdrawMax version 14.1.0.1212

### PR DESCRIPTION
**Pull request by PckgrMiranda**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/189540)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> No errors to post.

(Automated response - build 897.)

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by PckgrMiranda** [comment]
@PckgrBot

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> No errors to post.

(Automated response - build 897.)

File: manifests/e/EdrawSoft/EdrawMax/14.1.0.1212/EdrawSoft.EdrawMax.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ProductCode: '{037BAB81-3DF7-4381-A72C-A26B57C03548}_is1'
+ReleaseDate: 2024-08-22
+AppsAndFeaturesEntries:
+- DisplayName: '{cm:InstallAppName}(Build 14.0.0.1184)'
+  Publisher: EdrawSoft Co.,Ltd.
+  DisplayVersion: 14.0.0.1184
+  ProductCode: '{037BAB81-3DF7-4381-A72C-A26B57C03548}_is1'
+ElevationRequirement: elevatesSelf
+Installers:
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/e/EdrawSoft/EdrawMax/14.1.0.1212/EdrawSoft.EdrawMax.installer.yaml:25]:
```suggestion
- Architecture: x64
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @PckgrMiranda,

> Automatic Validation ended with: 
Edit: 
image

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by Trenly** [comment]
[Policy] Reset Labels

File: manifests/e/EdrawSoft/EdrawMax/14.1.0.1212/EdrawSoft.EdrawMax.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+FileExtensions:
+- eddx
+- edt
+- edx
+- vsd
+- vsdx
+ProductCode: '{037BAB81-3DF7-4381-A72C-A26B57C03548}_is1'
+ReleaseDate: 2024-08-22
+AppsAndFeaturesEntries:
+- DisplayName: '{cm:InstallAppName}(Build 14.0.0.1184)'
```
Comment (Trenly) [inline manifests/e/EdrawSoft/EdrawMax/14.1.0.1212/EdrawSoft.EdrawMax.installer.yaml:19]:
```suggestion
- DisplayName: 'Wondershare EdrawMax(Build 14.0.0.1184)'

```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by Trenly — APPROVED** [review]

**Comment by Trenly** [comment]
[Policy] Reset Feedback

**Review by Trenly — APPROVED** [review]

**Comment by Trenly** [comment]
@stephengillie - I think your review is still blocking this one for some reason

**Comment by stephengillie** [comment]
Hi @PckgrMiranda,

This manifest adds a DisplayVersion to the AppsAndFeaturesEntries that isn't present in previous manifest versions. This entry should be added to the previous versions, or removed from this version.

(Automated response - build 899.)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**